.vscode/launch.json

```diff
@@ -21,9 +21,11 @@
 			"runtimeExecutable": "${execPath}",
 			"args": [
 				"--extensionDevelopmentPath=${workspaceRoot}",
-				"--extensionTestsPath=${workspaceRoot}/out/test"],
+				"--extensionTestsPath=${workspaceRoot}/out/test",
+				"${workspaceRoot}/testFixture/"],
 			"stopOnEntry": false,
 			"sourceMaps": true,
+			"preLaunchTask": "npm: watch",
```

Comment:
can you explain the purpose of adding the watch here?

Reply:
actually I have no clue why that has been changed. Guess I will remove it.